examples/hybrid/driver.jl

```diff
@@ -117,7 +117,8 @@ end
 if config.parsed_args["check_conservation"]
     FT = Spaces.undertype(axes(sol.u[end].c.ρ))
     @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
-    @test sum(sol.u[1].c.ρe_tot) +
-          (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
-          sum(sol.u[end].c.ρe_tot) rtol = 100 * eps(FT)
+    @test sum(sol.u[1].c.ρe_tot) + (
+        p.radiation_mode.net_energy_flux_sfc[][] -
+        p.radiation_mode.net_energy_flux_toa[][]
```

Comment:
Would this cause a problem when running simulations without radiation? In any case, I think it would be better to put `net_energy_flux_sfc` and `net_energy_flux_toa` in the default cache, as they are used for the total energy flux at the surface and TOA. Currently, we only include radiative fluxes, but we should still have these two variables when running simulations without radiation.

Reply:
Okay, I thought that they were needed only for radiation. I'll revert this.